stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/message/internal/DatabaseMessageRepository.kt

```diff
@@ -174,8 +182,10 @@ internal class DatabaseMessageRepository(
     override suspend fun clear() {
         messageCache.evictAll()
         replyMessageCache.evictAll()
-        messageDao.deleteAll()
-        replyMessageDao.deleteAll()
+        scope.awaitWithMutex(dbMutex) {
```

Comment:
Same here, I think it's safer and more clearer to just use this here...

    override suspend fun clear() {
        dbMutex.withLock {
            messageDao.deleteAll()
            replyMessageDao.deleteAll()
        }
    }